iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -50,6 +52,11 @@ public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, T
         this.throwExceptionIfExtensionNotFound = throwExceptionIfExtensionNotFound;
     }
 
+    public ExportOptions(boolean withBranchSV, boolean indent, boolean onlyMainCc, TopologyLevel topologyLevel, boolean throwExceptionIfExtensionNotFound, String version) {
+        this(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound);
+        this.version = version;
+    }
```

Comment:
Implement this in the opposite way: the constructor with the biggest number of parameter should contain the logic. The other constructor(s) should call it with default values